libraries/Microsoft.Bot.Builder.Dialogs.Declarative/Types/UriTypeBinder.cs

```diff
@@ -1,42 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
-using Newtonsoft.Json.Serialization;
-
-namespace Microsoft.Bot.Builder.Dialogs.Declarative.Types
-{
-    public class UriTypeBinder : DefaultSerializationBinder
```

Comment:
>UriTypeBinder [](start = 17, length = 13)

This was only used to hook $type resolution.  Since we are 100% $kind now we don't need this anymore